clang/test/CIR/Lowering/types.cir

```diff
@@ -5,9 +5,9 @@
 module {
   cir.func @testTypeLowering() {
     // Should lower void pointers as opaque pointers.
-    %0 = cir.const(#cir.null : !cir.ptr<!void>) : !cir.ptr<!void>
+    %0 = cir.const(#cir.ptr<0> : !cir.ptr<!void>) : !cir.ptr<!void>
     // CHECK: llvm.mlir.null : !llvm.ptr
-    %1 = cir.const(#cir.null : !cir.ptr<!cir.ptr<!void>>) : !cir.ptr<!cir.ptr<!void>>
+    %1 = cir.const(#cir.ptr<0> : !cir.ptr<!cir.ptr<!void>>) : !cir.ptr<!cir.ptr<!void>>
```

Comment:
Also, is it possible to change printer/parser to pretty print to `#cir.ptr<null>` instead of `#cir.ptr<0>`? It'd feel more like C/C++ when reading CIR.

Have you given any thought on using `#cir.const_ptr` instead of `#cir.ptr`? I'm conflicted by which one we should use.

Reply:
Changed to `#cir.ptr<null>`